specs/regional_communities.md

```diff
@@ -1,61 +1,333 @@
+
+### Backwards Compatibility
 All proposals that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The doc must explain how to deal with these incompatibilities or the set of requirements to for the backward compatibility to be discussed in a submission. Submissions without a sufficient backwards compatibility treatise may be rejected outright.
 
+## Phases
+<!-- Do the project have multiple phases? Identify a high-level summary of each phase. -->
+
+### Phase 1: MVP
+- The app owner can create global parent communities for each Constellation
+- Users can find their community, assign them selves membership permissionlessly (similar to "follow")
+- Members can start posting messages to the Community board.
+- Members can perform a sibling split along one of the three dimensions
+- Members can perform a child split along one of the three dimensions, 
+- Member executing the split will become the owner of the new Community
+
+### Phase 2: Admin Functions
+- Admins can assign other admins
+- Users can request core memberships
+- Admins can approve core memberships
+- Admins can warn, silence and ban users
+
+### Phase 3: Enable voting
+- Members can request a vote on splitting the Community
+- Members can launch simple EasyPolls
+- Members can vote in a new core user
+- Members can vote to warn / silence / ban an offending user
+
+
 ## Use Cases
 
-### Actors
+## Actors
+<!-- List all collections that will use the widget. -->
+- Members
+   - Admin members
+   - Core members
+   - Home members
+   - Hangaround members
+- Non-members
+   - Non-member humans
+   - Non-member users
+
+## Actions
+<!-- List the actions each collective will take individually. -->
+
+### Find and Join a Community
+Finding a community aligned with my interests should be very easy. It's a core concept of this widget and serves to get people into open and collaborative communities rather than creating competing factions.
+
+Searching for communities should be possible through a few different ways
+- Search by map. Zoom into a map and update the list of available communities accordingly
+- Seach-as-you-type.
+   - Search by fuzzy search of geographies. Start typing in a city, country or region and all available options are shown. 
+   - Search by fuzzy search of constellations. Start typing in a constellation and all available options are shown.
+- Select you language, which updates the available communities. Default language is English.
+- Any combination of the above will further narrow the search
+
+
+### Approve a new Community member
+Approvind a new community member could be as simple as a "follow back" by the owner of the community or someone who has been given the keys to control the community account. 
+
+It is recommended that only human-verified accounts be allowed to become members, as a minimum requirement.
```

Comment:
I think we may get a push back of not allowing any account to become a member. I propose:
* community may optionally require only human-verified accounts for members. 
So, communities will have a choice if they are open for any accounts or only humans.

Reply:
Sure. I was going to leave it up to the developer, hackathon style, but we can be more prescriptive too.